plugins/wasi_nn/ggml.cpp

```diff
@@ -341,6 +350,8 @@ Expect<ErrNo> setInput(WasiNNEnvironment &Env, uint32_t ContextId,
   llama_context_params ContextParams = llama_context_default_params();
   ContextParams.n_ctx = GraphRef.CtxSize;
   ContextParams.n_batch = GraphRef.BatchSize;
+  ContextParams.n_threads = GraphRef.Threads;
+  ContextParams.n_threads_batch = GraphRef.Threads;
```

Comment:
Why is there a `n_threads_batch`? Is the value the same as the `n_threads`?

Reply:
`n_thread` is the number of threads to use during generation and `n_threads_batch` is the number of threads to use during batch and prompt processing. In llama.cpp:

```
  -t N, --threads N     number of threads to use during generation (default: 4)
  -tb N, --threads-batch N
                        number of threads to use during batch and prompt processing (default: same as --threads)
```

I think it is reasonable to limit both `n_threads` and `n_threads_batch` here.

Reply:
Okay.